### PR TITLE
chore(lte): Remove Make coverage targets for LTE

### LINKED ISSUE #13398: Consolidate coverage generation
Currently the MME coverage is generated by Make and all others by bazel. Can we do everything with bazel instead of splitting it up? This would make the agw-workflow much faster.

Blocked by #13399

- [x]  all places where AGW coverage is generated with make is replaced with bazel coverage generation
- [x]  create a new workflow and move coverage generation there ~agw-coverage.yml
  - [x]    use native build - other configs are not necessary
  - [x]    see if codecov actions can be upgraded
  - [x]    make sure the same codecov flags are used!
  - [x]    Unused Make targets should be removed

### PR DESCRIPTION
**Pull request by LKreutzer**
## Summary
- Clean up dead Make targets for LTE coverage
- Follow up to #14127
- Resolves #13398

## Test Plan

- Codecov for this PR https://app.codecov.io/gh/magma/magma/pull/14264

## Additional Information

- [ ] This change is backwards-breaking

### CODE REVIEW

**Review by ajahl — APPROVED** [review]

**Comment by LKreutzer** [comment]
Waiting for approval of approvers-agw-sessiond or until the group has been merged with another approvers group.

**Review by Neudrino — APPROVED** [review]

**Pull request merged**
